src/avfoundation.cs

```diff
@@ -13469,4 +13583,34 @@ interface AVCaptureMultiCamSession {
 		[Export ("systemPressureCost")]
 		float SystemPressureCost { get; }
 	}
+
+	[NoWatch, NoTV, Mac (10,15), iOS (13,0)]
+	[BaseType (typeof(AVMetadataObject))]
```

Comment:
add `[DisableDefaultCtor]` here since
* it's not used anywhere;
* the base type is abstract without an `init`
* we can remove it (it's not a breaking change) if we're wrong